src/modules/Spamchecker/Service.php

```diff
@@ -133,7 +133,7 @@ public function isInStopForumSpamDatabase(array $data)
     {
         $data['f'] = 'json';
         $url = 'https://www.stopforumspam.com/api?' . http_build_query($data);
-        $file_contents = $this->di['tools']->file_get_contents($url);
+        $file_contents = file_get_contents($url);
```

Comment:
I thought the plan was rewriting all file_get_contents to http module?

Reply:
https://github.com/FOSSBilling/FOSSBilling/pull/861

Reply:
100% the plan is long-term, however the reality is these functions will likely not survive 1-2months so I avoided doing too much at once. We need a much better spam handling module, integrating up to date CAPTCHA, etc.